### DISCUSSION TITLE
Ultimate NMZ plugin suggestion

### DISCUSSION THREAD

**Discussion by Jmac191** [runelite/runelite#16750] [Ideas]
I wasn't quite sure where to put this. But for the Ultimate NMZ Plugin, when the notifications pop up could we have an option to turn off the backdrop color? For example; Zapper, the color is set to purple, when a zapper spawns, the screen flashes purple/ fades in/ out or whatever you picked. Could we get an option to turn off purple and just have the zapper symbol show up. Sometimes it hurts my eyes when the screen flashes a solid color and we're being cave goblins with the lights out lol 
Anyways, thanks for your time.

**Comment by Bram91** [comment]
Support for hub plugins should be directed to the author of the plugin.
Get support for the Ultimate NMZ Plugin Hub plugin here: https://runelite.net/plugin-hub/show/ultimate-nmz.
You can also find the support link by searching for the plugin in the Plugin Hub panel and clicking the ? button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the Support menu option.

**Reply by Jmac191** [reply]
Thank you, I didn't know that.
